frontend/src/pages/clusterSettings/ClusterSettings.tsx

```diff
@@ -94,17 +103,27 @@ const ClusterSettings: React.FC = () => {
 
   React.useEffect(() => {
     setSettingsChanged(
```

Comment:
I think we can make `isSettingsChanged` to a const that use `React.useMemo` like
```
const isSettingsChanged = React.useMemo(() => !_.isEqual(clusterSettings, {
  pvcSize,
  cullerTimeout,
  userTrackingEnabled,
  notebookTolerationSettings,
}), [pvcSize, cullerTimeout, userTrackingEnabled, clusterSettings, notebookTolerationSettings])
```
And we can get rid of `setSettingsChanged`, correct me if I use this wrongly @andrewballantyne

Reply:
works. done.